src/lib/cache/subCache.cpp

```diff
@@ -799,6 +806,71 @@ void subCacheItemInsert
 
 /* ****************************************************************************
 *
+* subCacheItemInsert -
+*
+* NGSIv2 wrapper
+*
+*/
+void subCacheItemInsert
+(
+  const char*                        tenant,
+  const char*                        servicePath,
+  const ngsiv2::HttpInfo&            httpInfo,
+  const std::vector<ngsiv2::EntID>&  entities,
+  const std::vector<std::string>&    notifAttributes,
+  const std::vector<std::string>&    condAttributes,
+  const char*                        subscriptionId,
+  int64_t                            expiration,
+  int64_t                            throttling,
+  RenderFormat                       renderFormat,
+  bool                               notificationDone,
+  int64_t                            lastNotificationTime,
+  StringFilter*                      stringFilterP,
+  const std::string&                 status,
+  const std::string&                 q,
+  const std::string&                 geometry,
+  const std::string&                 coords,
+  const std::string&                 georel
+)
+{
+
+  NotifyConditionVector ncV;
+  EntityIdVector        enV;
+  AttributeList         attrL;
+
+  attrsStdVector2NotifyConditionVector(condAttributes, &ncV);
+  entIdStdVector2EntityIdVector(entities, &enV);
+  attrL.fill(notifAttributes);
```

Comment:
Here is where attrsStdVector2AttributeList is no longer used, as the functionality is covered by `AttributeList::fill()`